components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -926,6 +926,9 @@ def render_image_region(request, iid, z, t, conn=None, **kwargs):
             level = levels-int(zxyt[0])
             x = int(zxyt[1])*w
             y = int(zxyt[2])*h
+            # check if the image requires pyramid
+            if not img._re.requiresPixelsPyramid():
+                level = None
```

Comment:
Might be nicer to have this check happen in the BlitzGateway ```renderJpegRegion()``` so that BlitzGateway users benefit from it too?

Reply:
Need to check impact on method like ``renderBirdsEyeView``